pydis_site/apps/content/resources/guides/python-guides/windows/common-issues.md

```diff
@@ -0,0 +1,173 @@
+
+If you have installed python and are still having the issue, it is likely that you didn't check the "Add python.exe to
+PATH" checkbox on the first page of the installer. See [our guide on adding Python to PATH](../putting-python-on-path)
+for how to fix this. You can also use the `py` launcher instead of `python` by just replacing `python` in your command
+with `py`. See [this guide](../installing-and-using-python/#the-py-launcher) for more information on that.
+
+### I `pip` installed a package but when running my code I get a `ModuleNotFoundError`
+
+-   #### Are you actually getting a `ModuleNotFoundError`?
+
+    If you are using a code editor such as VSCode or PyCharm you may get a squiggly line under your import saying the
+    module couldn't be found, it is possible that this is just an mistake by the editor, so try to actually run your
+    code the way it is to ensure that it actually errors.
+
+    If your code runs fine, you could try restarting your editor. If you have newly installed a module it may just not
+    have detected it yet. If that doesn't help, you may need to configure your editor to ensure it is looking for the
+    module in the correct Python environment. See [our guide on virtual
+    environments](../installing-and-using-python/#virtual-environments) for more information.
+
+-   #### Was the install successful?
+
+    Look out for errors when installing the module you want. If you get an error, that's probably why the import isn't
+    working, so you should look into it. Often a Google search of the relevant error text will help.
+
+-   #### Did you use the correct module name?
+
+    Double check that you haven't made a typo in the name you are importing, or in what you installed from
+    [PyPI](https://pypi.org/), you need to make sure you type the name exactly as it should be.
+
+    Also, **the name you should `import` may not be the same as the name you `pip install`**. Check the module's
+    documentation or PyPI page if you are unsure. Examples of this are:
+
+    -   [**opencv-python**](https://pypi.org/project/opencv-python/): You need to `pip install opencv-python`, but the
+        import has to be `import cv2`
+    -   [**discord.py**](https://pypi.org/project/discord.py/): You need to `pip install discord.py`, but the import has
+        to be `import discord`
+    -   [**python-dotenv**](https://pypi.org/project/python-dotenv/): You need to `pip install python-dotenv`, but the
+        import has to be `import dotenv`
+    -   [**Pillow**](https://pypi.org/project/Pillow/): You need to `pip install Pillow`, but the import is
+        usually `from PIL import Image`.
+
+    Normal imports in Python can't have dashes (`-`) in them as it would be interpreted as subtraction. Though [there
+    are workarounds](https://stackoverflow.com/questions/7583652/python-module-with-a-dash-or-hyphen-in-its-name).
```

Comment:
Is this something you've seen people have an issue with? Is it with their own modules or installed ones?

Reply:
I think I've seen the issue a few times in a help channel, though can't recall exactly what was imported.

I do feel it's useful to explain why dashes (such a common character to name files/folder/packages with) don't work the same in Python syntax - even in imports.

Reply:
I don't think we should link to the workaround, as I don't think there's a case where that should be used and it will likely cause other problems.

We could mention that if a package from pypi has a hyphen in the name the actual import is probably different, and if they're trying to import a local file they should rename it.

Reply:
Alright, I can remove the workaround. I agree it's bad to send people towards unrecommended problem fixes.